python/tvm/auto_scheduler/measure.py

```diff
@@ -624,12 +647,7 @@ def local_build_worker(args):
         The build result of this Builder thread.
     """
     inp, build_func, timeout, verbose = args
-    if build_func == "default":
-        build_func = tar.tar
-    elif build_func == "ndk":
-        build_func = ndk.create_shared
-    else:
-        raise ValueError("Invalid build_func" + build_func)
+    build_func = BuildFunc.build_func
```

Comment:
Get `build_func` from the `args`, rather than get from the global `BuildFunc`.

Reply:
If this is a custom build func, we can not serialize it into args, because it is a callable object, not one str.

Reply:
custom build func is a python callable object, and it has attribute and so on, we can not serialize it into args.